### ISSUE TITLE
[css-variables] String identity of custom property name should be emphasized

### ISSUE THREAD

**Issue by aphillips** [w3c/csswg-drafts#7128]
4.1 Serializing Custom Properties
https://www.w3.org/TR/css-variables-1/#serializing-custom-props

> Custom property names must be serialized with the casing as provided by the author.
>
> Note: For non-custom properties, property names are restricted to the ASCII range and are [ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive), so implementations typically serialize the name lowercased.
>
> Specified values of [custom properties](https://www.w3.org/TR/css-variables-1/#custom-property) must be serialized exactly as specified by the author. Simplifications that might occur in other properties, such as dropping comments, normalizing whitespace, reserializing numeric tokens from their value, etc., must not occur.

Custom property names are not just restricted to the casing provided by the author, but also to the exact code point sequence. Unicode normalization of the name would potentially break matches elsewhere, since the requirement is _identical to_. This is covered for **values** (in the third paragraph above) but this should also be applied to the property name. I would suggest replacing this:

> Custom property names must be serialized with the casing as provided by the author.

with

> Custom property names must be serialized exactly as provided by the author without altering the case or Unicode normalization.

**Comment by tabatkins** [comment]
Note that in both names and values I'm referring solely to transformations/simplifications that currently occur in CSS. Nothing in CSS ever performs unicode normalization, and I'm somewhat hesitant to imply that it does.

**Comment by aphillips** [comment]
Fair enough. I18N's reaction here is that developers and authors leap at the code folding and forget the other variations text can endure (and which must not be altered when serializing). Since "File | Save As..." might be the result of a different (and potentially non-CSS) process, it is worth emphasizing. Perhaps to avoid invoking the demon of normalization:

> Custom property names must be serialized as the exact code point sequence provided by the author including not altering the case.

**Comment by tabatkins** [comment]
Ah, that sounds fine. Thanks!
